notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # TODO: Install GCPC from a release once prophet has been included in a release.
</pre><p>Making a note here to check the latest GCPC release.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>